python/nano/Windows_Development_Guide.md

```diff
@@ -0,0 +1,56 @@
+# Windows User Guide (WSL2)
+
+## Step 1: Install WSL2
+
+BigDL-Nano supports windows systems through *Windows Subsystem for Linux 2* (WSL2). 
```

Comment:
For Windows, we recommend using *Windows Subsystem for Linux 2* (WSL2) to run BigDL-Nano.